### PR TITLE
Improve 4.x doc

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: docs/getting_started/the_show_view.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+The Show View
+=============
+
+Now your admin allows you to create, edit and list blog posts and categories.
+But what if you just want to have a detailed view of one blog ? This chapter
```
Comment (jordisala1991) [inline docs/getting_started/the_show_view.rst:5]: the question mark without space

File: docs/reference/action_export.rst [hunk 2]
```diff
@@ -77,10 +95,4 @@ you can override the ``DataSourceInterface`` class::
...
 And then you can override the datasource set to the admin class.
 
-.. note::
-
-    **TODO**:
-    * customising the templates used to render the output
-    * publish the exporter documentation on the project's website and update the link
-
-.. _`the exporter bundle documentation`: https://github.com/sonata-project/exporter/blob/2.x/docs/reference/symfony.rst
+.. _`the exporter bundle documentation`: https://docs.sonata-project.org/projects/sonata-project-exporter/en/2.x/
```
Comment (jordisala1991) [inline docs/reference/action_export.rst:98]: it should be /projects/exporter/

File: docs/reference/action_list.rst [hunk 3]
```diff
@@ -763,27 +780,27 @@ query and displayed::
...
     {
-        $listMapper->add('thisPropertyDoesNotExist', null, [
+        $list->add('thisPropertyDoesNotExist', null, [
             'virtual_field' => true,
             'template' => 'path/to/your/template.html.twig'
         ]);
     }
 
-.. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/3.x/reference/list_field_definition
+.. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/4.x/reference/list_field_definition
```
Comment (jordisala1991) [inline docs/reference/action_list.rst:805]: last /

File: docs/getting_started/the_show_view.rst (hunk 1 above)
Comment (VincentLanglet) [reply docs/getting_started/the_show_view.rst:5]: Damned, french habits

**Comment by VincentLanglet** [comment]
Done @jordisala1991

**Review by jordisala1991 — DISMISSED** [review]

**Comment by VincentLanglet** [comment]
There was some inconsistency in the way menu, submenu, etc were used. I tried to improve this @jordisala1991

**Comment by VincentLanglet** [comment]
Things we can do
- There is some TODO in the doc
- The form_help_message.rst seems kinda useless.

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**